tests/integrationtest/static_route_test.go

```diff
@@ -138,13 +138,7 @@ func TestNodeAddNoPodCIDR(t *testing.T) {
 	g.Eventually(func() bool {
 		found, _ := objects.SharedAviGraphLister().Get(modelName)
 		return found
-	}, 10*time.Second).Should(gomega.Equal(true))
-	_, aviModel := objects.SharedAviGraphLister().Get(modelName)
-	g.Expect(aviModel.(*avinodes.AviObjectGraph).IsVrf).To(gomega.Equal(true))
-	nodes := aviModel.(*avinodes.AviObjectGraph).GetAviVRF()
-	g.Expect(len(nodes)).To(gomega.Equal(1))
-
-	g.Expect(len(nodes[0].StaticRoutes)).To(gomega.Equal(0))
+	}, 10*time.Second).Should(gomega.Equal(false))
```

Comment:
The VRF model will be present here, and the number of static routes will be empty since the CIDR is empty.

Reply:
VRF model mainly deals with populating static routes to controller. if there is no node with no static Route, AKO is not creating any VRF model to avoid memory consumption. So when this test is run, AKO doesn't have any valid node with CIDR. So we have taken out this conditions.